src/common/EmptyView/EmptyViewDouble.js

```diff
@@ -39,7 +39,8 @@ const EmptyViewDouble = ({
       marginTop: 40,
       fontSize: 26,
       fontWeight: 400,
-      lineHeight: '34px'
+      lineHeight: '34px',
+      hyphens: 'none'
```

Comment:
Please, add prefixes for more browser support. And also I'm suggesting improve `lang` tag to `lang="en"`, but I'm not sure it's necessary.